[Nightly] Newpipe Nightly 1143 crashing instantly upon opening it

### Checklist

- [x] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [x] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [x] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [x] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [x] This issue contains only one bug.
- [x] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).
- [x] I have read and understood the [AI policy](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#ai-policy). The content of this bug report is not generated by AI.

### Affected version

0.28.6-1143

### Steps to reproduce the bug

1. Open the latest build of NewPipe Nightly in any way you want.
2. Watch as the app tries to open.

### Expected behavior

NewPipe Nightly is able to open and function normally.

### Actual behavior

NewPipe Nightly crashes instantly upon opening it, to the point that the error handler has no time to fully start and display the error information. Any method of opening the app results in the crash occurring.

### Screenshots/Screen recordings

https://github.com/user-attachments/assets/5eaadca8-4104-430e-a70d-7b59b613e5a4

### Logs

The only way to get a log of this crash is with something that uses logcat; I used GrapheneOS's built-in log viewer to get the crash log. Relevant messages to the issue are near the end of the file.
[NewPipe nightly log d0cba707e674.txt](https://github.com/user-attachments/files/27869349/NewPipe.nightly.log.d0cba707e674.txt)

### Affected Android/Custom ROM version

GrapheneOS 16 (2026050901) Security Preview Release

### Affected device model

Google Pixel 8 Pro

### Additional information

I suspect that a commit made recently to the dev branch has caused this issue to occur with Nightly 1143, specifically one that changed something with PrettyTime. There is also no way to downgrade the app without losing all user data, at least on GrapheneOS (unless a downgrade release of Nightly is published), so this issue can result in data loss especially if the app was automatically updated without user interaction.

This issue also does not affect the latest stable version of NewPipe.

Here is the interesting part of the logcat:

```
java.lang.RuntimeException: Unable to create application org.schabi.newpipe.App
    at android.app.ActivityThread.handleBindApplication(ActivityThread.java:8189)
    at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2702)
    at android.os.Handler.dispatchMessage(Handler.java:132)
    at android.os.Looper.dispatchMessage(Looper.java:333)
    at android.os.Looper.loopOnce(Looper.java:263)
    at android.os.Looper.loop(Looper.java:367)
    at android.app.ActivityThread.main(ActivityThread.java:9332)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:566)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:837)
Caused by: java.util.MissingResourceException: Can't find bundle for base name org.ocpsoft.prettytime.i18n.Resources, locale en_US
    at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2110)
    at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1731)
    at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1629)
    at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1588)
    at java.util.ResourceBundle.getBundle(ResourceBundle.java:955)
    at org.ocpsoft.prettytime.impl.ResourcesTimeFormat.setLocale$1(r8-map-id-0c648f87f8fe0877f81762ed3b10c81557408b25493c6c2a8d3b76d41f3fb7bd:11)
    at org.ocpsoft.prettytime.PrettyTime.addUnit(r8-map-id-0c648f87f8fe0877f81762ed3b10c81557408b25493c6c2a8d3b76d41f3fb7bd:16)
    at org.ocpsoft.prettytime.PrettyTime.<init>(r8-map-id-0c648f87f8fe0877f81762ed3b10c81557408b25493c6c2a8d3b76d41f3fb7bd:27)
    at org.schabi.newpipe.util.Localization.resolvePrettyTime(r8-map-id-0c648f87f8fe0877f81762ed3b10c81557408b25493c6c2a8d3b76d41f3fb7bd:7)
    at org.schabi.newpipe.App.onCreate(r8-map-id-0c648f87f8fe0877f81762ed3b10c81557408b25493c6c2a8d3b76d41f3fb7bd:489)
    at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1406)
    at android.app.ActivityThread.handleBindApplication(ActivityThread.java:8184)
```

--- a/app/proguard-rules.pro
+++ b/app/proguard-rules.pro
@@ -54,3 +54,6 @@
 -keepclasseswithmembers class org.schabi.newpipe.** {
     kotlinx.serialization.KSerializer serializer(...);
 }
+
+# See https://github.com/TeamNewPipe/NewPipe/issues/13508
+-keep class org.ocpsoft.prettytime.i18n.Resources* { *; }

